### DISCUSSION TITLE
If I make changes to `copilot/app/manifest`, how do I push and apply the changes?

### DISCUSSION THREAD

**Discussion by lominming** [aws/copilot-cli#5798] [Q&A] [answered]
1. I already have an existing running app with `test` and `prod` using copilot
2. I already have CodePipeline setup
3. Now, I want to make changes to `copilot/app/manifest` (e.g. increase CPU for test)

What is the recommended way to push (e.g. CPU) changes to AWS? 
Can I simply just run `copilot svc deploy` to deploy the changes? Or do I have to delete first? Or do I have to do a `svc init` again?

**Comment by dannyrandall** [comment, marked as answer]
Hey @lominming! If you are using a Copilot pipeline, you can just push the manifest change to your repo, and the update should be applied by the pipeline. Otherwise, you can apply the change by doing `copilot svc deploy`.

**Comment by lominming** [comment]
@dannyrandall Can I also confirm that if I make changes to `copilot/pipelines/.../manifest.yml`, all I have to do is to simply push the file change to my repo? Do I have to run `copilot pipeline deploy`?

**Reply by iamhopaul123** [reply]
Hi @lominming. Any pipeline infra related changes (any changes for the pipeline manifest) would need you to run `copilot pipeline deploy`. Otherwise you can just commit and push the changes to the remote.
